src/main/java/frc/robot/subsystems/swervedrive/SwerveSubsystem.java

```diff
@@ -65,6 +69,10 @@ private SwerveSubsystem(File directory) {
                                              // angle.
 
     setupPathPlanner();
+
+    targetAngle = getHeading();
+    targetAngleController = new PIDController(5, 0, 0);
```

Comment:
There are a few hardcoded values in this PR (here, `maxSpeedMPS * 5` in SwerveDrive.java)- is this code meant to be merged as is?

Reply:
Moved the angle pid controller to the constants file. The 5 there is not currently used, but if we change stuff later it needs to be there because the modules need to go greater than the max speed to translate and rotate at the same time.